exchanges/bitstamp/bitstamp_test.go

```diff
@@ -26,7 +26,7 @@ const (
 	apiKey                  = ""
 	apiSecret               = ""
 	customerID              = "" // This is the customer id you use to log in
-	canManipulateRealOrders = false
+	canManipulateRealOrders = true
```

Comment:
revert so nobody accidently deploys funds to any position by accident

Reply:
done